rancher_pr_deploy.sh

```diff
@@ -55,14 +55,18 @@ rancher() {
     $@
 }
 
+yml2json() {
+  echo "$@" | docker run -a python:alpine -c 'import sys, yaml, json; json.dump(yaml.load(sys.stdin), sys.stdout)'
+}
+
 if [ "$TRAVIS" == true ] && [ "$TRAVIS_PULL_REQUEST" != false ] ; then
 
   fold_start "Generate_answers.txt"
   # POPULATES ANSWERS FILE WITH THE DEFAULTS FROM THE RANCHER-COMPOSE FILE
   RANCHER_CATALOG_ID=$( curl -su "${RANCHER_ACCESS_KEY}:${RANCHER_SECRET_KEY}" "${RANCHER_SERVER}/v1-catalog/templates/${RANCHER_CATALOG_NAME}:${RANCHER_TEMPLATE_NAME}" | jq --raw-output '.defaultTemplateVersionId' )
   curl -su "${RANCHER_ACCESS_KEY}:${RANCHER_SECRET_KEY}" "${RANCHER_SERVER}/v1-catalog/templates/${RANCHER_CATALOG_ID}" | \
   jq --raw-output '.files["rancher-compose.yml"]' | \
-    python -c 'import sys, yaml, json; json.dump(yaml.load(sys.stdin), sys.stdout)' | \
+    yml2json | \
```

Comment:
Finally, yml2json is broken. The -a flag needs STDIN/STDOUT and is redundent I think. Played around trying to get it to work for a bit but with no luck (the python yaml package needs installing in the container).